engine/apps/api/serializers/alert_receive_channel.py

```diff
@@ -200,20 +117,31 @@ def to_representation(cls, instance: AlertReceiveChannel) -> IntegrationAlertGro
         return {
             # todo: "inheritable" field is deprecated, remove in a future release.
             "inheritable": {label.key_id: True for label in instance.labels.all()},
-            "custom": cls._custom_labels_to_representation(instance.alert_group_labels_custom),
+            "custom": self._custom_labels_to_representation(instance.alert_group_labels_custom),
             "template": instance.alert_group_labels_template,
         }
 
-    @staticmethod
-    def _custom_labels_to_internal_value(
-        custom_labels: AlertGroupCustomLabelsAPI,
-    ) -> AlertReceiveChannel.DynamicLabelsConfigDB:
-        """Convert custom labels from API representation to the schema used by the JSONField on the model."""
+    def to_internal_value(
+        self,
+        validated_data: dict,
+    ) -> "AlertReceiveChannel.DynamicLabelsEntryDB":
```

Comment:
Is this type accurate?

Reply:
Good catch! It's a leftover from the old func, will fix it